packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -408,11 +408,16 @@ describe( 'Searching for a link', () => {
 		// fetchFauxEntitySuggestions resolves on next "tick" of event loop.
 		await eventLoopTick();
 
-		const searchResultTextHighlightElements = Array.from(
-			container.querySelectorAll(
-				'[role="listbox"] button[role="option"] mark'
-			)
-		);
+		const searchResults = within(
+			screen.getByRole( 'listbox', {
+				name: /Search results for.*/,
+			} )
+		).getAllByRole( 'option' );
+
+		const searchResultTextHighlightElements = Array.from( searchResults )
+			.map( ( result ) => result.querySelector( 'mark' ) )
+			.flat()
+			.filter( Boolean );
```

Comment:
We're changing the way we query for those `mark` elements to use screen queries.